### DISCUSSION TITLE
humidity, motion and magnetic sensors in STELLA 2.0 - commenting out chunks of code from code.py

### DISCUSSION THREAD

**Discussion by chadoulis** [STELLA-Landsat/STELLA#28] [Build Notes: How do I make a STELLA?]
I had to comment out the lines `140-142`, `201-205`, `240`, `243`, `245`, `246`, `260`, `261`, `285-288`, `300`, `301`, `303`, `306`, `307`, `309`, `370`, `373`, `376`, `379 ` from the `code.py` file because I think that STELLA 2.0 does not have any humidity, motion and magnetic sensors and I was getting 'Sensor not found' errors.

Was this the optimal way to resolve it? Are there plans for adding these sensors?

**Comment by paulmirel** [comment]
There are two ways to get the missing sensors, both of which require that the originally listed microcontroller return to stock: 
1. Replace the microprocessor with the one originally listed. This is difficult, but possible. It's an advanced desoldering task, difficult even with experience doing this sort of thing. 
2. Build an entirely new STELLA-2, with the microprocessor originally listed. 

The originally listed microprocessor, the Feather Bluefruit Sense, https://www.adafruit.com/product/4516, is out of stock at Adafruit, and also out of stock at Digi-Key, so neither option is currently available. 

It takes some electrical engineering work, and then some software engineering work, to add other sensors to the STELLA-2. Possible but not easy. We don't plan to do this, since the sensors will be on-board when the Feather Bluefruit Sense version of the microprocessor comes back into stock.

**Comment by paulmirel** [comment]
The Feather Bluefruit Sense microprocessor is available: 
https://www.electromaker.io/shop/product/adafruit-feather-nrf52840-sense
https://www.mouser.com/ProductDetail/Adafruit/4516
